master/buildbot/status/buildstep.py

```diff
@@ -355,7 +370,9 @@ def setProcessObjects(self, build, master):
 
     def upgradeToVersion1(self):
         if not hasattr(self, "urls"):
-            self.urls = {}
+            self.urls = []
```

Comment:
@BarryUnity before we put the feature in staging we need to fix the versioning of the object since this will rewrite the information stored in disk